runtime-modules/content/src/tests/curators.rs

```diff
@@ -71,93 +71,94 @@ fn curator_group_management() {
+        //             ContentModerationAction::DeleteChannel,
+        //             ContentModerationAction::HideChannel,
+        //             ContentModerationAction::HideVideo,
+        //             ContentModerationAction::ChangeChannelFeatureStatus(
+        //                 PausableChannelFeature::ChannelFundsTransfer,
+        //             ),
+        //             ContentModerationAction::ChangeChannelFeatureStatus(
+        //                 PausableChannelFeature::CreatorCashout,
+        //             ),
+        //             ContentModerationAction::ChangeChannelFeatureStatus(
+        //                 PausableChannelFeature::CreatorTokenIssuance,
+        //             ),
+        //             ContentModerationAction::ChangeChannelFeatureStatus(
+        //                 PausableChannelFeature::ChannelUpdate,
+        //             ),
+        //             ContentModerationAction::ChangeChannelFeatureStatus(
+        //                 PausableChannelFeature::VideoCreation,
+        //             ),
+        //             ContentModerationAction::ChangeChannelFeatureStatus(
+        //                 PausableChannelFeature::VideoNftIssuance,
+        //             ),
+        //             ContentModerationAction::ChangeChannelFeatureStatus(
+        //                 PausableChannelFeature::VideoUpdate,
+        //             ),
+        //         ]),
+        //     ),
+        // ]);
 
-        // Update group permissions
-        assert_ok!(Content::update_curator_group_permissions(
-            Origin::signed(LEAD_ACCOUNT_ID),
-            curator_group_id,
-            permissions.clone()
-        ));
+        // // Update group permissions
+        // assert_ok!(Content::update_curator_group_permissions(
+        //     Origin::signed(LEAD_ACCOUNT_ID),
+        //     curator_group_id,
+        //     permissions.clone()
+        // ));
 
-        // Check CuratorGroupPermissionsUpdated event
-        assert_eq!(
-            System::events().last().unwrap().event,
-            Event::Content(RawEvent::CuratorGroupPermissionsUpdated(
-                curator_group_id,
-                permissions.clone()
-            ))
-        );
+        // // Check CuratorGroupPermissionsUpdated event
+        // assert_eq!(
+        //     System::events().last().unwrap().event,
+        //     MetaEvent::content(RawEvent::CuratorGroupPermissionsUpdated(
+        //         curator_group_id,
+        //         permissions.clone()
+        //     ))
+        // );
 
         // Validate permissions
-        let group = Content::curator_group_by_id(curator_group_id);
-        assert_eq!(group.get_permissions_by_level().len(), 3);
+        // let group = Content::curator_group_by_id(curator_group_id);
+        // assert_eq!(group.get_permissions_by_level().len(), 3);
         // Iterate over privilege levels from 0 to 3
         // (3 will be a "non-existent map entry" case)
```

Comment:
Comments should also be commented-out:
```suggestion
        // // Iterate over privilege levels from 0 to 3
        // // (3 will be a "non-existent map entry" case)
```